deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py

```diff
@@ -441,3 +441,81 @@ def check_invalid_provider_update_from_year_log(provider):
                 error_message="ERROR: Invalid content for 'update_from_year' option at module 'vulnerability-detector'",
                 prefix=r".*"
             )
+
+
+def check_baseline_scan_log(agent_id='000'):
+    """Check that the baseline scan is logged correctly.
+
+    Args:
+        agent_id (str): Agent id.
+    """
+    check_vuln_detector_event(
+                timeout=vd.VULN_DETECTOR_SCAN_TIMEOUT,
```

Comment:
Please, check and use the following format

```
def function(param1, param2,
             param3, param4)
```

Instead of

```
def function(
    param1, param2,
    param3, param4)
```